util/error/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "ethcore-error"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+
+[dependencies]
+rlp = { path = "../rlp" }
+ethcore-bigint = { path = "../bigint" }
+snappy = { path = "../snappy" }
+error-chain = "0.11.0-rc.2"
```

Comment:
it would be good to use a single version of `error-chain`. this pr uses 2 different (`0.11` and `0.11-rc2`)

Reply:
Only version `0.11.0-rc.2` is used. I believe `0.11.0` is auto generated in Cargo.lock which will just drop the suffix `rc.2`.

Reply:
u are right!